lib/akami/wsse/signature.rb

```diff
@@ -151,6 +193,12 @@ def body_digest
         Base64.encode64(OpenSSL::Digest::SHA1.digest(body)).strip
       end
 
+      def timestamp_digest
+        return nil unless timestamp
+        timestamp = canonicalize(at_xpath(@document, "//Envelope/Header/Security/Timestamp"))
+        Base64.encode64(OpenSSL::Digest::SHA1.digest(timestamp)).strip
```

Comment:
if `timestamp` is nil, which it can be, an exception is raised.  Should this line be ended with `... if timestamp`?